guides/doc-Release_Notes/topics/foreman-3.5.3.adoc

```diff
@@ -0,0 +1,27 @@
+= Foreman 3.5.3
+
+A full list of changes is available on https://projects.theforeman.org/issues?set_filter=1&sort=id%3Adesc&status_id=closed&f%5B%5D=cf_12&op%5Bcf_12%5D=%3D&v%5Bcf_12%5D%5B%5D=1696[Redmine]
+
+== Foreman
+
+=== Unattended Installations
+
+* preseed_netplan_generic_interface with DHCP interface - https://projects.theforeman.org/issues/35578[#35578]
+
+=== API
+
+* Parameter \'search\' on fact_value API endpoint results in internal server error - https://projects.theforeman.org/issues/35990[#35990]
```

Comment:
The script does something wrong with quotes, because I never found out how you properly escape `'` in asciidoc. Sometimes you need to escape it, sometimes you don't.

```suggestion
* Parameter 'search' on fact_value API endpoint results in internal server error - https://projects.theforeman.org/issues/35990[#35990]
```